packages/salesforcedx-utils-vscode/src/cli/commandBuilder.ts

```diff
@@ -9,11 +9,13 @@ export class Command {
   public readonly command: string;
   public readonly description?: string;
   public readonly args: string[];
+  public readonly logName?: string;
```

Comment:
Expanding the command builder to support adding log name when we build the command.